.github/workflows/tests.yml

```diff
@@ -0,0 +1,19 @@
+name: Tests
+
```

Comment:
### Optional
- [ ] It would be better if we improve our commit messages, for example; instead of `worked` write `Work` instead keeping in mind to capitalize our commit messages :100: